webapi/src/main/scala/org/knora/webapi/app/ApplicationActor.scala

```diff
@@ -516,28 +517,25 @@ class ApplicationActor extends Actor with Stash with LazyLogging with AroundDire
 
         var msg =
             """
-              | _   __                              ___  ______ _____
-              || | / /                             / _ \ | ___ \_   _|
-              || |/ / _ __   ___  _ __ __ _ ______/ /_\ \| |_/ / | |
-              ||    \| '_ \ / _ \| '__/ _` |______|  _  ||  __/  | |
-              || |\  \ | | | (_) | | | (_| |      | | | || |    _| |_
-              |\_| \_/_| |_|\___/|_|  \__,_|      \_| |_/\_|    \___/
+              |  ____  ____  ____         _    ____ ___
+              | |  _ \/ ___||  _ \       / \  |  _ \_ _|
+              | | | | \___ \| |_) |____ / _ \ | |_) | |
+              | | |_| |___) |  __/_____/ ___ \|  __/| |
+              | |____/|____/|_|       /_/   \_\_|  |___|
```

Comment:
I'm confused about these names. I thought "Knora" and "DSP-API" were the same thing. Personally I think "Knora" is a better name than "DSP-API" (takes too long to say and is impossible to remember).

Reply:
Knora is slowly phased out. The official name of the platform is DSP, hence DSP-API.

Reply:
I think DSP is a terrible name. Do a Google search for "DSP" and see what you get. Can the whole team have a discussion about this?

Reply:
Sure, we can do it next Tuesday after the Daily Scrum.